processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryQueryToolChestTest.java

```diff
@@ -97,6 +98,8 @@ public class GroupByQueryQueryToolChestTest extends InitializedNullHandlingTest
   public static void setUpClass()
   {
     NullHandling.initializeForTests();
+    //noinspection ResultOfObjectAllocationIgnored
+    new AggregatorsModule();
```

Comment:
hmm, we should move the complex serde registrations of that module to a static method it calls instead of doing this so its more obvious what is going on here

Reply:
cool, ill do that. It's weird that we need to call this in the first place, I would have expected the stuff to be registered. I'll make it more explicit and add a comment.

Reply:
nit: forget about this?

Reply:
oops, yeah. Will fix it up